content/articles/creating-an-animated-javafx-field-validation-alert/index.md

```diff
@@ -0,0 +1,761 @@
+
+![Project view in scene builder](/engineering-education/creating-an-animated-javafx-field-validation-alert/project-view-in-scene-builder.png "Project view in scene builder")
+
+Run the application. It opens a new Java window of the application.
+In it, there is a button that when clicked returns a new text in the window.
+The image below shows the output:
+
+![Running the first project](/engineering-education/creating-an-animated-javafx-field-validation-alert/running-the-first-project.png "Running the first project")
+
+Close the application for the next steps.
+
+### Create the Login form
+
+#### Modify the Scene
+
+Open the `HelloApplication.java` file. In it, perform the following:
+
+- change the Title of the window to 'Login Form'.
+- Ensure that the width of the scene is 400 by 600 pixels.
+- Add an icon in the title tab of the JavaFX project. Download the `icon.png` file [here](https://github.com/justusmbuvi/Creating-an-Animated-JavaFX-field-Validation-alert/raw/main/src/main/resources/authentication/login/icon.png).
+
+These are shown below:
+
+```java
+package authentication.login;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class HelloApplication extends Application {
+    @Override
+    public void start(Stage stage) throws IOException {
+        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("hello-view.fxml"));
+        
+        // Set the scene width and height to 400 and 600 respectively
+        Scene scene = new Scene(fxmlLoader.load(), 400, 600);
+        
+        // Set the name of the window to 'Login Form'
+        stage.setTitle("Login Form");
+        
+        // Add an Icon to the title bar
+        stage.getIcons().add(new Image(HelloApplication.class.getResourceAsStream("icon.png")));
+        stage.setScene(scene);
+        stage.show();
+    }
+
+    public static void main(String[] args) {
+        launch();
+    }
+}
+```
+
+#### Design the view
+
+Open up the `hello-view.fxml` file using the Scene Builder.
+Do the following in it:
+
+Delete the VBOX and the element in it.
+Do this by right-clicking on it and selecting delete in the left tab.
+
+Drag and drop an AnchorPane where the VBox was deleted.
+It loads a new AnchorPane that holds other controls in it.
+
+The AnchorPane will have the following:
+
+- Style
+- - `-fx-text-fill`:  #8ecae6
+- Layout
+- - `Pref Width`: 400
+- - `Pref Height`: 600
+
+The layout makes it fit perfectly with the scene initially created.
+
+Drag and drop a label from the controls tab into the AnchorPane in the center of the SceneBuilder.
+
+The label will have the following:
+
+- Properties
+- - `Text`: LOGIN
+- - `Font`: System 25px Bold
+- Style
+- - `-fx-text-fill`:  #219ebc
+- Layout
+- - `Pref Width`: 100
+- - `Pref Height`: 20
+- - `Layout X`: 150
+- - `Layout Y`: 40

```

Comment:
You could write these properties in your code to avoid redundancy and having a long article.

Reply:
inside the code, its hard for one to easily follow up since the reader will have to search it out instead of just inputting it directly